package.json

```diff
@@ -14,7 +14,10 @@
     "react-ui/node_modules"
   ],
   "dependencies": {
-    "express": "^4.14.1"
+    "express": "^4.14.1",
+    "react": "^15.5.4",
```

Comment:
I don't think these react packages are needed in the server package.json file. Or am I missing something?

Reply:
They don't need to be included on the server side.